modules/update-common-terms.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: REFERENCE
+[id="update-common-terms_{context}"]
+
+= Common Terms
+
+Control Plane:: The _control plane_, which is composed of control plane machines, manages the {product-title} cluster. The control plane machines manage workloads on the compute machines, which are also known as worker machines. The cluster itself manages all updates to the machines by the actions of the Cluster Version Operator (CVO), the Machine Config Operator (MCO), and a set of individual Operators.
+
+Cluster Version Operator:: The _Cluster Version Operator_ (CVO) checks with the OpenShift Update Service to view valid updates and update paths based on your current component versions and information in the graph.
+
+Machine Config Operator:: The _Machine Config Operator_ (MCO) is a cluster-level operator that manages the operating system and keeps the cluster up to date and configured. Through the MCO, platform administrators can configure and update systemd, CRI-O and Kubelet, the kernel, NetworkManager, and other system features on the nodes.
++
+For more information, see link:https://docs.openshift.com/container-platform/latest/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks[Machine config overview].
+
+Edge:: _Edges_ are shown on the graph and they are versions that you can safely update to.
```

Comment:
I like "edge" for folks who are already thinking about [updates as a graph][1].  But for folks at the "I'm interested in updates" stage, who may not have yet realized that they might have several possible targets as choices, I'd rather we use something with more context like "recommended updates" or "update targets" or some such.  I'm fine with "recommended update edges" or similar to mix context from both the sysadmin-with-a-cluster and update-graph contexts, but I think the most important word to include in this name is "update".

[1]: https://github.com/openshift/cincinnati-graph-data